src/Illuminate/Console/Scheduling/ScheduleWorkCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Illuminate\Console\Scheduling;
+
+use Illuminate\Console\Command;
+
+class ScheduleWorkCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'schedule:work';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Start the schedule worker';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $this->info('Schedule worker started successfully.');
+
+        while (true) {
+            if (now()->second === 0) {
```

Comment:
Not tested, but I think this won't work in Lumen. Lumen has the console component, but not `now()` helper.

Reply:
Yeah this probably needs to use the `\Illuminate\Support\Carbon` class directly.

Reply:
Thanks @Vusys!

Reply:
I've updated the PR!